### PR TITLE
[ENH] Fix various typos

### PR DESCRIPTION
**Pull request by jsheunis**
This PR fixes various typos that were tracked in https://github.com/datalad-handbook/book/issues/733. Closes https://github.com/datalad-handbook/book/issues/733.

### CODE REVIEW

File: docs/basics/101-115-symlinks.rst [hunk 1]
```diff
@@ -13,7 +13,7 @@ As the text file is stored in Git and not git-annex, no content unlocking is nec
 As we saw within the demonstrations of :command:`datalad run`, modifying content of non-text files, such as ``.jpg``\s, typically requires the additional step of *unlocking* file content, either by hand with the :command:`datalad unlock` command, or within :command:`datalad run` using the ``-o``/``--output`` flag.
 
 There is one detail about DataLad datasets that we have not covered yet.
-Its both a crucial aspect to understanding certain aspects of a dataset, but it is also a potential source of confusion that we want to eradicate.
+It's both a crucial aspect to understanding certain aspects of a dataset, but it is also a potential source of confusion that we want to eradicate.
```
Comment (mslw) [inline docs/basics/101-115-symlinks.rst:16]: I think we can use this opportunity and make this sentence read better by doing something like this
```suggestion
It is a crucial component to understanding certain aspects of a dataset, but it is also a potential source of confusion that we want to eradicate.
```

Comment (mslw) [reply docs/basics/101-115-symlinks.rst:16]: Other than that, a merge is long overdue :)

**Review by mslw — APPROVED** [review]

**Comment by mslw** [comment]
The RTD build failed due to something that got since changed in master (error starts with "Obtaining autorunrecord from git+git://..." and newer PRs use "git+https://..." instead) so it's in fact good to go.

**Comment by adswa** [comment]
ooops, sorry for missing this and thanks a lot for proposing and merging!

**Pull request merged**